src/shared/components/annotation/CancerHotspots.tsx

```diff
@@ -48,15 +48,22 @@ export default class CancerHotspots extends React.Component<ICancerHotspotsProps
         );
     }
 
-    public static title(isHotspot:boolean, is3dHotspot:boolean)
+    public static title(isHotspot:boolean, is3dHotspot:boolean, count?: number, customInfo?: JSX.Element)
     {
-        const recurrentHotspot = isHotspot ? (<b>Recurrent Hotspot</b>) : "";
-        const maybeAnd = isHotspot && is3dHotspot ? "and" : "";
-        const clusteredHotspot = is3dHotspot ? (<b>3D Clustered Hotspot</b>) : "";
+        const recurrentHotspot = isHotspot ? (<b>Recurrent Hotspot</b>) : undefined;
+        const maybeAnd = isHotspot && is3dHotspot ? <span>and</span> : undefined;
+        const clusteredHotspot = is3dHotspot ? (<b>3D Clustered Hotspot</b>) : undefined;
+
+        let countInfo: JSX.Element|undefined;
+
+        if (count) {
+            const sample = count > 1 ? "samples" : "sample";
+            countInfo = <span><b>{count}</b> {sample} with</span>;
+        }
 
         return (
             <span>
-                {recurrentHotspot} {maybeAnd} {clusteredHotspot}
+                {countInfo} {recurrentHotspot} {maybeAnd} {clusteredHotspot} {customInfo}
```

Comment:
if you're using <span> only to be able to return a set of children, you can now use <React.Fragment>